plutus-tx/src/PlutusTx/List.hs

```diff
@@ -60,387 +61,397 @@ import Prelude (Maybe (..), (.))
 -- | Plutus Tx version of 'Data.List.uncons'.
 uncons :: [a] -> Maybe (a, [a])
 uncons = \case
-    []   -> Nothing
-    x:xs -> Just (x, xs)
-{-# INLINABLE uncons #-}
+  [] -> Nothing
+  x : xs -> Just (x, xs)
+{-# INLINEABLE uncons #-}
 
 -- | Test whether a list is empty.
 null :: [a] -> Bool
 null = \case
-    [] -> True
-    _  -> False
-{-# INLINABLE null #-}
+  [] -> True
+  _ -> False
+{-# INLINEABLE null #-}
 
 length :: [a] -> Integer
 length = go
-  where
-    go = \case
-      [] -> 0
-      _:xs -> Builtins.addInteger 1 (go xs)
-{-# INLINABLE length #-}
-
--- | Plutus Tx version of 'Data.List.map'.
---
---   >>> map (\i -> i + 1) [1, 2, 3]
---   [2,3,4]
---
+ where
+  go = \case
+    [] -> 0
+    _ : xs -> Builtins.addInteger 1 (go xs)
+{-# INLINEABLE length #-}
+
+{-| Plutus Tx version of 'Data.List.map'.
+
+  >>> map (\i -> i + 1) [1, 2, 3]
+  [2,3,4]
+-}
 map :: forall a b. (a -> b) -> [a] -> [b]
 map f = go
-  where
-    go :: [a] -> [b]
-    go = \case
-        []   -> []
-        x:xs -> f x : go xs
-{-# INLINABLE map #-}
+ where
+  go :: [a] -> [b]
+  go = \case
+    [] -> []
+    x : xs -> f x : go xs
+{-# INLINEABLE map #-}
 
 -- | Returns the conjunction of a list of Bools.
 and :: [Bool] -> Bool
 and = \case
-    []     -> True
-    x : xs -> if x then and xs else False
-{-# INLINABLE and #-}
+  [] -> True
+  x : xs -> if x then and xs else False
+{-# INLINEABLE and #-}
 
 -- | Returns the disjunction of a list of Bools.
 or :: [Bool] -> Bool
 or = \case
-    []     -> False
-    x : xs -> if x then True else or xs
-{-# INLINABLE or #-}
+  [] -> False
+  x : xs -> if x then True else or xs
+{-# INLINEABLE or #-}
 
 -- | Determines whether any element of the structure satisfies the predicate.
 any :: forall a. (a -> Bool) -> [a] -> Bool
 any f = go
-  where
-    go :: [a] -> Bool
-    go = \case
-        []     -> False
-        x : xs -> if f x then True else go xs
-{-# INLINABLE any #-}
+ where
+  go :: [a] -> Bool
+  go = \case
+    [] -> False
+    x : xs -> if f x then True else go xs
+{-# INLINEABLE any #-}
 
 -- The pragma improves some of the budget tests.
```

Comment:
I wonder which pragma this comment is referring to....

Reply:
https://github.com/IntersectMBO/plutus/commit/b751871d7ef2e773fcec389e02c0d9013d66e8a7#diff-8fb42f388e5687122fb4b268f866c3e24ef5ffe54603c172810113b55ce14cf1R110 

 it points to `all`